### ISSUE TITLE
The trained model is not so useful than vgg-m in vot2017 using ECO structure?

### ISSUE THREAD

**Issue by he010103** [egundogdu/CFCF#1]
I guessed the trained model is finetuned by vgg-m. But when I replace the vgg-m by your model in ECO. The EAO in vot2017 is not so good as vgg-m. Is there any other parameter setting when using your model?

**Comment by egundogdu** [comment]
Hi,
Yes, the fine-tuned model is pre-trained VGG-M-2048 on ImageNet. ECO is an improved version of CCOT with some major enhancements to further increase the accuracy and the runtime. For this purpose, they reduce the feature dimension. I suspect it might not be good to reduce the number of feature dimensions for our fine-tuned features. Second, please note that the number of CG iterations has been reduced to 1 instead of 5 of CCOT. Another point is the fine-tuned model is more invariant to appearance changes of the object, thus the effective learning rate of ECO (I think) should be investigated to obtain further increase with our fine-tuned features.

In summary, since I have not experienced ECO yet, I suggest you perform component analysis on ECO with CFCF features, such as deactivating the factorization operator (dimensionality reduction), different learning rate and CG values, and deactivating the sample space model. If I find out an interesting result, I will report it here.

Thank you for this discussion.

**Comment by he010103** [comment]
The default CG_iter is 5 in ECO. Besides, I test ECO with only CNN features without PCA, which can have performance of 39.05 in vot2016 with original VGG-m. However, I donot try to run without sample space model, maybe it works. Thanks for your reply!
